feeder/parsers/PT.py

```diff
@@ -0,0 +1,44 @@
+import requests, dateutil, arrow
+from bs4 import BeautifulSoup
+
+COUNTRY_CODE = 'PT'
+
+def fetch_PT():
+
+	r = requests.get('http://www.centrodeinformacao.ren.pt/EN/InformacaoExploracao/Pages/EstatisticaDiaria.aspx')
+	soup = BeautifulSoup(r.text, 'html.parser')
```

Comment:
It's an html parser: text -> tree! AFAIK it's the most used html parser for Python.